### PR TITLE
Connect Four Game

### LINKED ISSUE #1415: Connect Four Game
**Project title**
Connect Four Game

**Project Description**
I will add the Connect Four Game project made using HTML, CSS and JS to the repo.

**Stack**:  
And in order to tick the check box just put x inside them for example - [x] like this. Please delete options that are not relevant.

- [x] Html
- [x] CSS
- [x] JavaScript

### PR DESCRIPTION
**Pull request by SarthakSKumar**
<hr>

## Description 📜

Connect Four Game

Fixes #1415 

<hr>

## Type of change 📝

Please delete options that are not relevant. In order to tick the check box just put x inside them for example [x] like this

- [x] Add a new project

<hr>

## Domain of Contribution 📊

Please delete options that are not relevant. In order to tick the check box just put x inside them for example [x] like this

- [x] Project using HTML, CSS & JS

<hr>
 
## Checklist ✅

Please delete options that are not relevant. In order to tick the check box just put x inside them for example [x] like this

- [x] I have performed a self-review of my own code or work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Recommended) 📸
image
image

### CODE REVIEW

File: ConnectFour/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## How to set-up:
+
+- Clone this repo
+
+- Open it in any Text Editor
+
+- Enjoy it !!!
+
+![ConnectFour](https://user-images.githubusercontent.com/68684482/124650666-98456c00-deb7-11eb-80e7-87e999d8ab96.png)
```
Comment (Mrjoy832) [inline ConnectFour/README.md:23]: add a working demo video

**Comment by SarthakSKumar** [comment]
Shall make the required changes and update. Thanks

File: ConnectFour/README.md (hunk 1 above)
Comment (Mrjoy832) [reply ConnectFour/README.md:23]: add demo video...not SS only

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

**Comment by Mrjoy832** [comment]
## @SarthakSKumar finish by tomorrow either I will close

**Review by gurjeetsinghvirdee — APPROVED** [review]

**Pull request merged**